packages/client/tests/functional/invalid-env/tests.ts

```diff
@@ -1,7 +1,11 @@
 import testMatrix from './_matrix'
 
 // @ts-ignore this is just for type checks
-declare let prisma: import('@prisma/client').PrismaClient
+type PrismaClient = import('@prisma/client').PrismaClient
+// @ts-ignore this is just for type checks
+declare const newPrismaClient: NewPrismaClient<typeof PrismaClient>
+// @ts-ignore
+declare let Prisma: typeof import('@prisma/client').Prisma
```

Comment:
Maybe we could add a comment about what these variables are for?

Reply:
That needs to be documented once somewhere, duplicating the comment in each test file is not very practical.

Reply:
On the other hand, we do often duplicate "this is just for type checks" comments from the example test together with the `// @ts-ignore` directive. Perhaps we could replace them with a brief but useful description of each variable?

Reply:
(that would be out of scope of this PR though, IMO)

Reply:
I think what we could do is instead of providing them as globals, provide them as a first argument to `setupTestSuite`, for example:

```ts

// @ts-ignore
testMatrix.setupTestSuite<typeof import('@prisma/client')>((generated, suiteConfig, suiteMeta) => {
// generated.Prisma, generated.client and generated.newPrismaClient are available here
})
```

And `@ts-ignore` can go away once we start generating client for development as well.
I agree that's out of scope of this PR, created a separate issue for that: #14322